exp/api/v1alpha3/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha3 contains API Schema definitions for the exp v1alpha3 API group
+// +kubebuilder:object:generate=true
+// +groupName=exp.cluster.x-k8s.io
+package v1alpha3
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "exp.cluster.x-k8s.io", Version: "v1alpha3"}
```

Comment:
Should this be in it's own group? If we lump everything together in the same experimental api group, then all experimental types would need to be versioned in lockstep.

Reply:
That's certainly something that crossed my mind while working on this, although that complicates by a lot not only the release process but also the amount of kubebuilder sub-projects we might introduce.

For the time being, I'd like to stick with a single experimental group, keep things simple until we have we learn more about how the experiments will grow

Reply:
I suppose my main concern here is that if we don't tackle it now, then it will be more difficult to tackle later, especially if we allow experiments to be added and/or bump type versions for existing experiments after the v0.3.0 release.